### PR TITLE
chore: display running webpack version during tests

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**

chore

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
NA

**Summary**
This display running webpack version during tests, useful to check if the CI cache is working correctly and locally to ensure we're running tests on correct env.

**Does this PR introduce a breaking change?**
No

**Other information**
/cc @evilebottnawi

### CODE REVIEW

**Review by evilebottnawi — APPROVED** [review]
Thanks

**Comment by evilebottnawi** [comment]
Let's see versions on CI and merge if it will be fine

**Review by evenstensberg — DISMISSED** [review]

**Review by rishabh3112 — CHANGES_REQUESTED** [review]
Need CI fix. `--progress=profile` test is failing.

**Comment by anshumanv** [comment]
yep wip

**Review by rishabh3112 — DISMISSED** [review]

File: scripts/globalSetup.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+const { version } = require('webpack');
+
+module.exports = () => console.log(`Running tests for webpack @${version}`);
```
Comment (snitin315) [inline scripts/globalSetup.js:3]: Can we improve its visibility? 

Screenshot at 2020-11-03 13-28-39

```suggestion
module.exports = () => console.log(`Running tests for webpack @${version}`);
```
```suggestion
module.exports = () => console.log(`\n Running tests for webpack @${version} \n`);
```

**Review by snitin315 — APPROVED** [review]

**Review by rishabh3112 — APPROVED** [review]

**Pull request merged**